services/dask-sidecar/docker/boot.sh

```diff
@@ -39,7 +39,7 @@ if [ ${DASK_START_AS_SCHEDULER+x} ]; then
   echo "$INFO" "Starting as ${SCHEDULER_VERSION}..."
   if [ "${SC_BOOT_MODE}" = "debug-ptvsd" ]; then
 
-    exec watchmedo auto-restart --recursive --pattern="*.py" -- \
+    exec watchmedo auto-restart --recursive --pattern="*.py;*/src/*" --ignore-patterns="*test*;pytest_simcore/*;setup.py;*ignore*" --ignore-directories -- \
```

Comment:
question, is this going to restart if I'm changing the libraries it uses? (it's a general question for all the other services as well)

Reply:
yes.

this is monitoring everything below ``/devel`` folder, which binds all the packages as well.